modules/mobile/app/services/mobile/v0/vaccinations/service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Vaccinations
+      # Service that connects to CDC's vaccinations XML table
+      # https://www2.cdc.gov/vaccines/iis/iisstandards/XML.asp?rpt=vax2vg
+      #
+      class Service
```

Comment:
Could this logic either move to a background job or be called from a background job? That way it can be scheduled to run once a day. Schedules are set in the [sidekiq_scheduler.yml](https://github.com/department-of-veterans-affairs/vets-api/blob/00f79f734be15295546d877f495335c647b16690/config/sidekiq_scheduler.yml#L226-L229) file. And then the job would be in a similar location (but in our module) to this [batch job](https://github.com/department-of-veterans-affairs/vets-api/blob/5b880ce76c1e33d6923cf6b6eaf23617fc571980/modules/covid_vaccine/app/jobs/covid_vaccine/scheduled_batch_job.rb).

Reply:
That's a good point. I was planning to put it on the scheduler, but I didn't think about the fact that scheduled jobs should be async.